src/scene_server.py

```diff
@@ -7,19 +7,36 @@
 from fastapi import FastAPI
 from lynx.common.object import Object
 from lynx.common.scene import Scene
+from lynx.common.enitity import Entity
 from pydantic import BaseModel
 
 
 def execution_runtime(pipe: AioConnection, object_id: int):
```

Comment:
What do you think about moving `execution_runtime` to a separate file?

Reply:
Sure, I just did not want to waste time on that